frontend/src/lib/components/EditableField/EditableField.tsx

```diff
@@ -63,6 +65,12 @@ export function EditableField({
 
     const isSaveable = !minLength || tentativeValue.length >= minLength
 
+    const mouseDownOnCancelButton = (e: React.MouseEvent): void => {
+        // if saveOnBlur is set the onBlur handler of the input fires before the onClick event of the button
+        // this onMouseDown handler fires before the input can see the click and fire onBlur
+        e.preventDefault()
+    }
```

Comment:
Calling `preventDefault` on a `mouseDown` event on a button prevents the `onBlur` on a loosely related input from running...? 🤔 

Seems like sorcery, but it works when I tested it, so 🍨